Extension/src/LanguageServer/extension.ts

```diff
@@ -922,8 +923,8 @@ function reportMacCrashes(): void {
             const crashObject: Record<string, string> = {};
             if (err?.code) {
                 // If the directory isn't there, we have a problem...
-                crashObject["fs.stat: err.code"] = err.code;
-                telemetry.logLanguageServerEvent("MacCrash", crashObject, undefined);
+                crashObject["stat: errCode"] = err.code;
```

Comment:
Sorry if I was unclear before.  Please change `"stat: errCode"` to `"errorCode"`.  I don't like to have spaces or special characters in telemetry property names.